fe1-web/src/features/digital-cash/components/DigitalCashWalletItems.tsx

```diff
@@ -1,39 +1,24 @@
-import { CompositeScreenProps, useNavigation } from '@react-navigation/core';
-import { StackScreenProps } from '@react-navigation/stack';
 import { ListItem } from '@rneui/themed';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { View } from 'react-native';
 
 import { PoPIcon } from 'core/components';
-import { AppParamList } from 'core/navigation/typing/AppParamList';
-import { WalletParamList } from 'core/navigation/typing/WalletParamList';
 import { Hash } from 'core/objects';
 import { Color, Icon, List, Typography } from 'core/styles';
 import STRINGS from 'resources/strings';
 
 import { DigitalCashHooks } from '../hooks';
-import { DigitalCashFeature } from '../interface';
-
-type NavigationProps = CompositeScreenProps<
-  StackScreenProps<WalletParamList, typeof STRINGS.navigation_wallet_home>,
-  StackScreenProps<AppParamList, typeof STRINGS.navigation_app_lao>
->;
 
 const DigitalCashWalletItems = ({ laoId }: IPropTypes) => {
   const balance = DigitalCashHooks.useTotalBalance(laoId);
 
-  const navigation = useNavigation<NavigationProps['navigation']>();
-
   // the digital cash wallet item should be at the top
   const listStyles = List.getListItemStyles(true, false);
 
   return (
     <>
-      <ListItem
-        containerStyle={listStyles}
-        style={listStyles}
-        onPress={() => navigation.navigate(STRINGS.navigation_wallet_digital_cash_wallet)}>
+      <ListItem containerStyle={listStyles} style={listStyles} onPress={() => {}}>
```

Comment:
onPress -> do nothing ?

Reply:
Whoops, good catch! This component should have been deleted!